lib/util/Testable.js

```diff
@@ -7,7 +7,12 @@ var Testable = {
 
     request: request,
 
-    requestPromise: Promise.promisify(request)
+    // wawjr3d 5/31/2016 TODO adding multiArgs to support bluebird 2.
+    //  Remove second parameter and update all uses of requestPromise
+    //  when support for bluebird 3 is dropped
```

Comment:
https://github.com/petkaantonov/bluebird/issues/307 has all the details. tl;dr is bluebird 2 promisify automatically made an array when function returned multiple values. bluebird 3 only passes the first return value